packages/frontend/.env.production

```diff
@@ -0,0 +1,2 @@
+
+REACT_APP_FIREBASE_CONFIG='{"apiKey":"AIzaSyCWgxNPREB1Pw7JXaOjq5c63TdhK0HKHcg","authDomain":"flat-split.firebaseapp.com","projectId":"flat-split","storageBucket":"flat-split.appspot.com","messagingSenderId":"404672973040","appId":"1:404672973040:web:496005cc4f28f56ab75a35","measurementId":"G-D05DK4ZWJ2"}'
```

Comment:
Maybe we should not commit this firebase config to an open source repository as it contains sensitive information like api keys. Can we just add this information into secret part?

Reply:
I believe this is public? @frasermcc9 are you able to weigh in here?

Reply:
Its not secret, and hiding it would be needless obscurity. These values are sent to anyone visiting our website and are substituted into the normal js at build time.

Reply:
ok, just keep it